web/src/providers/Wizard/Wizard.provider.tsx

```diff
@@ -61,6 +64,16 @@ const WizardProvider = ({children, stepsMap}: IProps) => {
     WizardAnalytics.onStepComplete(activeStepId);
   }, [activeStepId, isFinalStep, updateWizard, wizard.steps]);
 
+  const onCompleteStep = useCallback(
+    async (stepId: TWizardStepId) => {
+      if (WizardService.shouldUpdate(stepId, wizard.steps)) {
+        const updatedSteps = WizardService.completeStep(stepId, wizard.steps);
+        await updateWizard({steps: updatedSteps}).unwrap();
```

Comment:
Could you add the analytics event here? 
`WizardAnalytics.onStepComplete(activeStepId);`